transformer/request.py

```diff
@@ -26,6 +26,7 @@ class HttpMethod(enum.Enum):
     PUT = enum.auto()  #: PUT
     OPTIONS = enum.auto()  #: OPTIONS
     DELETE = enum.auto()  #: DELETE
+    PATCH = enum.auto()  #: DELETE
```

Comment:
The doc. comment here should read `#: PATCH` (also makes sense to put this new enum right after `PUT`).

Reply:
oops, done.